packages/react/src/components/Dashboard/DashboardGrid.test.e2e.jsx

```diff
@@ -1162,6 +1162,7 @@ DashboardAllCardsAsResizable.propTypes = {
 describe('DashboardGrid', () => {
   beforeEach(() => {
     MockDate.set(1537538254000);
+    cy.viewport(1680, 900);
```

Comment:
Why are we setting the viewport to the same thing before and after all?

Reply:
I've realized cypress loses this value often. That's why in some of the automated tests items will get pushed off screen until you resize, so I've started adding it to tests as I come across it to keep the viewport rendering correctly. It's probably overkill to have it in both before/after here though.